### DISCUSSION TITLE
Insert core blocks without inner blocks?

### DISCUSSION THREAD

**Discussion by ghost** [WordPress/gutenberg#28719] [Q&A] [answered]
Hey,

I recently wondered if that is possible in some way? E.g., I'd like to insert a Heading block into my custom block without using the Innerblocks component.

Thanks!

**Comment by ellatrix** [comment, marked as answer]
Hi @dnnsjsk! That's not possible since `InnerBlocks` is the mechanism to insert blocks into another block. What stops you from using `InnerBlocks`? If you wish to only insert one block, it's possible to lock the template: https://developer.wordpress.org/block-editor/tutorials/block-tutorial/nested-blocks-inner-blocks/
